orcid-persistence/src/main/resources/db/updates/org_unique_constraints_add_disambiguated_org.xml

```diff
@@ -0,0 +1,27 @@
+<!--
+
+    =============================================================================
+
+    ORCID (R) Open Source
+    http://orcid.org
+
+    Copyright (c) 2012-2014 ORCID, Inc.
+    Licensed under an MIT-Style License (MIT)
+    http://orcid.org/open-source-license
+
+    This copyright and license information (including a link to the full license)
+    shall be included in its entirety in all copies or substantial portion of
+    the software.
+
+    =============================================================================
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">	
+	            
+	<changeSet id="change_org_unique_constraint" author="George Nash" dbms="postgresql">
+		<sql>ALTER TABLE org DROP CONSTRAINT org_unique_constraints;</sql>
+		<sql>ALTER TABLE org ADD CONSTRAINT org_unique_constraints UNIQUE (name,city,region,country,org_disambiguated_id);</sql>
```

Comment:
Hey @bobcaprice,

Why are you adding the **org_disambiguated_id**  to the unique constraint?

Thanks

Reply:
I talked with @rcpeters about it, it is good to go!